src/components/user-card/UserCard.js

```diff
@@ -15,3 +15,24 @@ import React from 'react';
 //  этот div будет содержать:
 //  - блок h4 с классом card-title, он должен вывести имя и фамилию (first_name, last_name)
 //  - блок div c классом "card-text", содержащий 2 div блока для поля email и поля address
+
+export const UserCard = (props) => {
+    const {user} = props;
+    if (!user) return null;
+    const {first_name, last_name, email, address, _links} = user;
+    return (
+        <div className='may-user-card card'>
+            <img src = {_links.avatar.href} className='may-user-card-avatar rounded-circle' alt='user photo'/>
```

Comment:
you can do deeper destructuring
    const {first_name, last_name, email, address, _links : {avatar: { href }}} = user;

and then write just like this
            <img src = {href} className='may-user-card-avatar rounded-circle' alt='user photo'/>

it is not a 'must have'
but again, that is just looking a bit better

Reply:
I corrected it